src/qt/unlimitedmodel.cpp

```diff
@@ -50,20 +50,34 @@ void UnlimitedModel::Init()
     // Ensure restart flag is unset on client startup
     setRestartRequired(false);
 
+    unsigned int ead = excessiveBlockSize;
```

Comment:
I think this is at best a very badly named variable (ead), or a bug - we should NEVER be assigning an excessive  block size to a variable that could be confused with having to do with Accept Depth.
They are two different things (EB and AD), and should never be confused, esp. not in variable names.

I am also not a friend of short variable names unless they are limited to extremely short code sections / loop indexes. But this is the minor consideration, since always easy to refactor.